### ISSUE TITLE
Unit test failing on v1.0 branch

### ISSUE THREAD

**Issue by rouke-broersma** [stryker-mutator/stryker-net#1517]
**Describe the bug**
Unit test MutantOrchestratorTests.ShouldNotAddReturnDefaultToEnumerationMethods is failing due to what seem to be whitespace issues.

**Logs**
```csharp
Shouldly.ShouldAssertException : isSame
should be
True
but was
False

Additional Info:
AST's are not equivalent. Line[11]
actual:if(StrykerNamespace.MutantControl.IsActive(0)){;}else{            yield return value;
expect:            if(StrykerNamespace.MutantControl.IsActive(0)){;}else{
Actual(full):
using System;
using System.Collections.Generic;
using System.Text;
namespace StrykerNet.UnitTest.Mutants.TestResources
{
class TestClass
{public static IEnumerable<object> Extracting<T>(this IEnumerable<T> enumerable, string propertyName)
{
foreach (var o in enumerable)
{
if(StrykerNamespace.MutantControl.IsActive(0)){;}else{            yield return value;
}        }
}
public static IEnumerable<object> Extracting<T>(this IEnumerable<T> enumerable)
{
if(StrykerNamespace.MutantControl.IsActive(1)){;}else{        yield break;
}      }}}
, expected:
using System;
using System.Collections.Generic;
using System.Text;
namespace StrykerNet.UnitTest.Mutants.TestResources
{
class TestClass
{public static IEnumerable<object> Extracting<T>(this IEnumerable<T> enumerable, string propertyName)
{
foreach (var o in enumerable)
{
if(StrykerNamespace.MutantControl.IsActive(0)){;}else{
yield return value;
}
}
}
public static IEnumerable<object> Extracting<T>(this IEnumerable<T> enumerable)
{
yield break;
}}}
```

**Additional context**
https://dev.azure.com/stryker-mutator/Stryker/_build/results?buildId=7118&view=results

**Comment by sbergen** [comment]
In addition to the whitespace issue on the first diff (line 11), there's also a problem with what the test is testing:
```
public static IEnumerable<object> Extracting<T>(this IEnumerable<T> enumerable)
{
if(StrykerNamespace.MutantControl.IsActive(1)){;}else{        yield break;
}      }}}
```

at the end.

**Comment by sbergen** [comment]
I was not quite on point with my previous comment, the second change is due to #1472 not being taken into account. #1507 seems to be the cause for the whitespace breakage (reverting it fixed the test).

But wether or not this means that the test data should just be updated, or if code changes are required is beyond me. Hope this helps :)

**Comment by rouke-broersma** [comment]
This should not have happened... No clue why the pipeline didn't catch this before merge.

**Comment by sbergen** [comment]
To clarify, v1.0 broke at 17bf601, when master was merged in. So I'm saying merging #1507 into v1.0 was the issue. I'm not sure of the details of how your pipelines are set up, but one way to prevent this is always running tests against a merged version of the target and base branch, and always merging branches through PRs (including "special" branches like v1.0 and master).

**Comment by sbergen** [comment]
If making pipelines work with the merged state is out of the scope, you can also do
```
git checkout v1.0
git checkout -b merge-master
git merge master
```
and open a PR out of that (I didn't find a PR like that, sorry if I missed it).

**Comment by rouke-broersma** [comment]
Mhm I think we forgot to set the merged state requirement on v1 when we made that our main branch. Fixed that now thank you! This was already set-up on master..

**Comment by dupdob** [comment]
> I was not quite on point with my previous comment, the second change is due to #1472 not being taken into account. #1507 seems to be the cause for the whitespace breakage (reverting it fixed the test).

Yes, that is the source of the problem. #1507 is a fix for master, and #1472 is only present in V1.
The test code needs to be amended to match the actual value (in V1 branch, not master)

**Comment by rouke-broersma** [comment]
Fixed
